src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Square extends Figure {
```

Comment:
The class name 'Square' is appropriate as it is a noun, but remember not to start the class implementation with an empty line.